### PR TITLE
Alpha4 Changes

### PR DESCRIPTION
**Pull request by v-dharmarajv**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by bharat-kalyan-namburi — APPROVED** [review]

File: sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,54 @@
 
 ### Other Changes
 
+## 1.4.0-beta.1 (2024-11-22)
```
Comment (fhaghbin-msft) [inline sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md:13]: (2024-11-22) is the same date for 1.3.0 line 24?

Comment (fhaghbin-msft) [reply sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md:13]: Also do we have InterruptAudioAndAnnounce api here? and all new events for hold and play?

Comment (v-vdharmaraj) [reply sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md:13]: > (2024-11-22) is the same date for 1.3.0 line 24?

Yes GA4 and Beta5 are released on the same date.

Comment (v-vdharmaraj) [reply sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md:13]: > Also do we have InterruptAudioAndAnnounce api here? and all new events for hold and play?

we have the generated model changes, but needs to be updated in the SDK but will do it another PR.

**Review by fhaghbin-msft — APPROVED** [review]

**Pull request merged**